flink-libraries/flink-table/src/test/scala/org/apache/flink/table/expressions/SqlExpressionTest.scala

```diff
@@ -156,6 +156,8 @@ class SqlExpressionTest extends ExpressionTestBase {
     // testSqlApi("('hello world', 12)", "hello world") // test base only returns field 0
     testSqlApi("ARRAY[TRUE, FALSE][2]", "false")
     testSqlApi("ARRAY[TRUE, TRUE]", "[true, true]")
+    testSqlApi("MAP['k1', 'v1', 'k2', 'v2']['k2']", "v2")
```

Comment:
Please also update the documentation for both Table API and SQL. They need to be kept consistent.